src/app_core.py

```diff
@@ -68,6 +69,7 @@ def strdef(name, default):
 
 app.config["DEBUG"] = boolify(os.getenv("DEBUG"))
 app.config["TESTNET"] = boolify(os.getenv("TESTNET"))
+app.config["USE_SENGRID"] = boolify(os.getenv("USE_SENDGRID"))
```

Comment:
```suggestion
app.config[MAIL_SENDGRID_API_KEY] = os.getenv("SENDGRID_API_KEY")
app.config["USE_SENGRID"] = boolify(os.getenv("SENDGRID_API_KEY")))
```

and remove `set_vital_setting("SENDGRID_API_KEY", "MAIL_SENDGRID_API_KEY")` below

Reply:
make the changes as above but using "USE_SENDGRID" instead of "USE_SENGRID".

Reply:
using test_email to send emails when using sendgrid gives me the following:
```
[app_core ERROR] Exception on /test_email [POST]
Traceback (most recent call last):
  File "/app/src/email_utils.py", line 51, in send_email_sendgrid
    sg = SendGridAPIClient(app.config["MAIL_SENDGRID_API_KEY"])
KeyError: 'MAIL_SENDGRID_API_KEY'
```

Reply:
found another typo.... made the changes and this appears to work now.